types/recharts/index.d.ts

```diff
@@ -228,9 +229,9 @@ export interface DotProps {
 }
 export class Dot extends React.Component<DotProps, {}> {}
 
-export interface ErrorBarProps {
+export interface ErrorBarProps extends Partial<CSSStyleDeclaration> {
 	dataKey?: string | number;
-	width?: number;
+	// width?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?